packages/sdk-react/src/components/payables/PayableDetails/PayableDetailsForm/PayableDetailsForm.tsx

```diff
@@ -329,6 +329,7 @@ const PayableDetailsFormBase = forwardRef<
                             }
                           >
                             {counterpartBankAccountsToSelect(
+                              // @ts-expect-error - we have to fix this
```

Comment:
I assume that the hook `packages/sdk-react/src/core/queries/useCouterpartsBankAccounts.ts` is also a part of this task's scope. If you refactored this hook, this problem should be fixed.

Reply:
this is what I meant
https://github.com/team-monite/monite-sdk/pull/163/commits/6e253540b3c332679d332d47d950617144bc9332